Makefile

```diff
@@ -12,24 +9,6 @@ clean:
 	$(RM) -r *.egg-info/ build/ dist/ rpm/
 	$(RM) ../rpm_s3_mirror_* test-*.xml
 
-rpm:
```

Comment:
Was it on purpose that we do not have a mke target to build an rpm anymore?

Reply:
Yes. RPM building will be done downstream. Also, RPM building was not done properly anyway, proper way of RPM building is to do it per OS distribution.